content/courses/onchain-development/intro-to-onchain.md

```diff
@@ -11,133 +11,192 @@ description:
 ## Summary
 
 - **Onchain programs** are programs that run on Solana.
-- Programs have one or more **instruction handlers**. Instruction handlers are
-  functions that process the **instructions** in Solana transactions.
-- Instruction handlers write their data to Solana **accounts** and then either
-  succeed or fail.
-- Solana programs are most commonly written in **Rust**, typically with the
-  **Anchor** framework.
-- Anchor programs have IDLs, which are small JSON files describing programs.
-  IDLs can be used to automatically create JS/TS libraries for a program.
+- Programs contain one or more **instruction handlers**. These are functions
+  that process **instructions** within Solana transactions.
+- Instruction handlers write data to Solana **accounts** and either succeed or
+  fail.
+- Solana programs are commonly written in **Rust** with the **Anchor**
+  framework.
+- Anchor programs have IDLs (Interface Description Language) and small JSON
+  files that describe programs. IDLs can be used to automatically generate JS/TS
+  libraries for a program.
 
 ## Overview
 
-Each Solana cluster (`mainnet-beta`, `testnet`, `devnet`, `localnet`) is
-effectively a single computer with a globally synchronized state. The programs
-that run on Solana - the ones that create tokens, swap tokens, art marketplaces,
-escrows, market makers, DePIN apps, auctions, retail payments platforms, etc -
-are called **Solana apps**.
+Each Solana cluster (`mainnet-beta`, `testnet`, `devnet`, `localnet`) functions
+as a single computer with a globally synchronized state. The programs that
+operate on Solana—creating tokens, swapping tokens, powering art marketplaces,
+escrows, market makers, DePIN apps, auctions, retail payment platforms, and
+more—are referred to as **Solana apps**.
```

Comment:
```suggestion
Each Solana cluster (`mainnet-beta`, `testnet`, `devnet`) functions
as a single computer with a globally synchronized state. The programs that
operate on Solana—creating tokens, swapping tokens, powering art marketplaces,
escrows, market makers, DePIN apps, auctions, retail payment platforms, and
more—are referred to as **Solana apps**.
```

Reply:
Technically localnet isn't a cluster.

Reply:
I have merged the main branch and removed localnet from the list.